pkg/util/chunk/iterator.go

```diff
@@ -58,6 +58,14 @@ func NewIterator4Slice(rows []Row) Iterator {
 	return &Iterator4Slice{rows: rows}
 }
 
+// NewIterator4SliceNoInterface returns a Iterator4Slice.
+func NewIterator4SliceNoInterface(rows []Row) Iterator4Slice {
+	return Iterator4Slice{
+		rows:   rows,
+		cursor: 0,
+	}
+}
+
```

Comment:
I think modify NewIterator4Slice return Iterator4Slice is ok.  No new functions need to be added. See NewIterator4Chunk.

Reply:
> I think modify NewIterator4Slice return Iterator4Slice is ok. No new functions need to be added. See NewIterator4Chunk.

okk